buku

```diff
@@ -36,11 +36,11 @@ import threading
 import time
 from typing import Any, Dict, List, Optional, Tuple, Union
 import webbrowser
+import pyreadline
```

Comment:
Why do we need this? You are already importing pyreadline as readline if readline import fails.

Reply:
my bad. Sorry.